settings/spartan7.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# Copyright (C) 2017-2021  The Project X-Ray Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
+source $(dirname ${BASH_SOURCE[0]})/spartan7_50.sh
```

Comment:
I think this should already contain the `spartan7_50.sh` content without sourcing it, and the `spartan7_50.sh` should be removed.

Reply:
I deleted the old `spartan7.sh` and renamed `spartan7_50.sh` to `spartan7.sh`, deleted `spartan7_50` from the Makefile and changed the Kokoro config files as well. Let's see if it works this time!